cdap-hbase-compat-1.2-cdh5.7.0/pom.xml

```diff
@@ -50,6 +50,13 @@
   </dependency>
   <dependency>
     <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-hbase-compat-base</artifactId>
```

Comment:
All the hbase-compat modules should have a dependency on `cdap-hbase-compat-base`, isn't it? It is currently coming from the transitive dependency through data-fabric, but it is better to always declare the direct dependency that this module is actually depending upon.